vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -90,6 +90,9 @@ type VerticalPodAutoscalerSpec struct {
 	// resources for all containers in the pod, without additional constraints.
 	// +optional
 	ResourcePolicy *PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,3,opt,name=resourcePolicy"`
+
+	// Denote the recommender to be used
+	RecommenderName []string `json:"recommenderName,omitempty" protobuf:"bytes,1,opt,name=recommenderName"`
```

Comment:
for the api part, I am thinking we can leave more complexity here like defining recommender strategies for multiple recommenders, but that can be left for future use cases.

Reply:
Yes, I agree. If we will later contribute more recommendation algorithms to the default recommender, we can use recommenderName as a field to select one among candidate algorithms.